content/source/docs/cloud/integrations/service-now/admin-guide.html.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: "cloud"
+page_title: "Administrator Guide - ServiceNow Service Catalog Integration -
+Terraform Cloud and Terraform Enterprise"
+description: |-
+  Administrator guide for the ServiceNow Service Catalog Integration.
+---
+
```

Comment:
Page needs an H1 and a sentence explaining what it's for.

Reply:
In particular, needs to orient the reader about why they would do this type of flows-and-actions customization as opposed to just creating catalog items as described in the service catalog page.